Conan.VisualStudio/Services/InfoBarEventsHandler.cs

```diff
@@ -22,8 +22,8 @@ public void OnClosed(IVsInfoBarUIElement infoBarUIElement)
 
         public void OnActionItemClicked(IVsInfoBarUIElement infoBarUIElement, IVsInfoBarActionItem actionItem)
         {
-            var projectGuid = _vcProjectService.UnloadProject(_project);
-            _vcProjectService.ReloadProject(projectGuid);
+            //var projectGuid = _vcProjectService.UnloadProject(_project);
+            //_vcProjectService.ReloadProject(projectGuid);
```

Comment:
Do we want to keep these lines commented here?

Reply:
no, but entire file is removed in https://github.com/conan-io/conan-vs-extension/pull/129 anyway